### PR TITLE
Override ossec_rules.xml to disable netstat check

### PR DESCRIPTION
**Pull request by conorbranagan**
This includes a [netstat check](https://github.com/ossec/ossec-hids/blob/master/etc/rules/ossec_rules.xml#L149) that uses a good chunk of CPU on hosts that have many active connections. We were seeing spikes up to 60-80% CPU which hurts the main service running on machines.

### CODE REVIEW

**Comment by alq666** [comment]
@conorbranagan instead of this could you have [this file](https://github.com/ossec/ossec-hids/blob/master/etc/rules/ossec_rules.xml#L149) as a `cookbook_file` with the netstat bit commented out?

File: templates/default/ossec.conf.erb [hunk 1]
```diff
@@ -94,7 +94,6 @@
     <include>vmware_rules.xml</include>
     <include>ms_dhcp_rules.xml</include>
     <include>asterisk_rules.xml</include>
-    <include>ossec_rules.xml</include>
```
Comment (alq666) [inline templates/default/ossec.conf.erb:97]: A bit heavy-handed to remove it altogether.

**Comment by dougdep** [comment]
beat me to it @alq666 would rather just comment that line out. I assume those rules get pulled down during install?

**Comment by devolur** [comment]
I am fine with @conorbranagan doing whatever needs to be done to fix this issue, including removing the rule.

**Review by alq666 — APPROVED** [review]

**Comment by alq666** [comment]
@conorbranagan 👍

**Pull request merged**